sponsorship_sub_management/__manifest__.py

```diff
@@ -36,9 +36,7 @@
     "license": "AGPL-3",
     "website": "https://www.compassion.ch",
     "depends": [
-        "sponsorship_compassion",
-        "base_automation",
```

Comment:
The module is used with 3 automated actions
- Sponsorship SUB Proposal
- Sponsorship SUB Proposal validation
- Check SUB Accept/Reject after 50 days

I think we need these for the module to work correctly, and the automated actions are depending on base_automation right?

Reply:
Module sponsorship_compassion is already depending on base_automation through recurring_contract, so we don't need to specify again the dependency.